src/AlgebraicGeometry/ToricVarieties/cohomCalg/auxiliary.jl

```diff
@@ -18,7 +18,7 @@ function command_string(v::NormalToricVarietyType, c::Vector{ZZRingElem})
     
     # Add information about the Stanley-Reisner ideal to string_list
     current_coordinate_names = [string(x) for x in Hecke.gens(cox_ring(v))]
-    new_coordinate_names = ["x$i" for i = 1:length(current_coordinate_names)]
+    new_coordinate_names = [Symbol("x$i") for i = 1:length(current_coordinate_names)]
```

Comment:
```suggestion
    new_coordinate_names = [Symbol(:x, i) for i = 1:length(current_coordinate_names)]
```
otherwise this still allocates a string